DepartmentDetails/src/com/fsd/group4/departmentinterfaces/DepartmentUtilities.java

```diff
@@ -0,0 +1,12 @@
+package com.fsd.group4.departmentinterfaces;
```

Comment:
Can we follow naming convetion for defining an interface?

Interface names should be capitalized like class names e.g. interface RasterDelegate;
interface Storing;